streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBKeyValueStoreTest.java

```diff
@@ -39,7 +40,7 @@
             Deserializer<K> keyDeser = (Deserializer<K>) context.keyDeserializer();
             Serializer<V> valSer = (Serializer<V>) context.valueSerializer();
             Deserializer<V> valDeser = (Deserializer<V>) context.valueDeserializer();
-            supplier = Stores.create("my-store").withKeys(keySer, keyDeser).withValues(valSer, valDeser).persistent().build();
+            supplier = Stores.create("my-store").withKeys(Serializations.serialization(keySer, keyDeser)).withValues(Serializations.serialization(valSer, valDeser)).persistent().build();
```

Comment:
Why are we still passing serializer and deserializers around separately?  One of the main reasons for this PR is to merge both into a single serde, no?